webbpsf/__init__.py

```diff
@@ -13,119 +13,42 @@
-        + "an integer pixel for the grid spacing in the PSF calculation.",
-    )
+        STPSF Functionality is currently the same as WebbPSF, only the name is different.
+        WebbPSF is now an alias of STPSF and is running code from the STPSF library.
 
-    default_output_mode = _config.ConfigItem(
-        "both",
-        "Should output include the oversampled PSF, a copy rebinned onto the integer detector spacing, or both?\
-        Options: 'oversampled','detector','both' ",
-    )
-    default_fov_arcsec = _config.ConfigItem(
-        5.0, "Default field of view size, in arcseconds per side of the square "
-    )
+        !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
 
-    # Should be package settings:
-    WEBBPSF_PATH = _config.ConfigItem(
-        "from_environment_variable",
-        "Directory path to data files required for WebbPSF calculations, such as OPDs and filter transmissions.\
-        This will be overridden by the environment variable $WEBBPSF_PATH, if present.",
-    )
-    autoconfigure_logging = _config.ConfigItem(
-        False,
-        "Should WebbPSF configure logging for itself and POPPY? This adds handlers that report "
-        "calculation progress and information",
-    )
-    logging_level = _config.ConfigItem(
-        ["INFO", "DEBUG", "WARN", "ERROR", "CRITICAL", "NONE"],
-        # (the default value is the first item in the options list)
-        "Desired logging level for WebbPSF optical calculations.",
-    )
-    logging_filename = _config.ConfigItem(
-        "none", "Desired filename to save log messages to."
-    )
-    logging_format_screen = _config.ConfigItem(
-        "[%(name)7s] %(message)s", "Format for lines logged to the screen."
-    )
-    logging_format_file = _config.ConfigItem(
-        "%(asctime)s [%(name)s:%(levelname)s] %(filename)s:%(lineno)d: %(message)s",
-        "Format for lines logged to a file.",
+        """,
+        DeprecationWarning,
+        stacklevel=2,  # Shows the webbpsf import code
     )
+    _warned = True
 
+sys.modules["webbpsf"] = stpsf
 
-conf = Conf()
-
-from . import utils  # noqa - must go after config
-from . import trending  # noqa - must go after config
-from .utils import setup_logging, restart_logging, system_diagnostic, measure_strehl  # noqa - must go after config
-
-from poppy import (  # noqa
-    display_psf,
-    display_psf_difference,
-    display_ee,
-    measure_ee,
-    display_profiles,
-    radial_profile,
-    measure_radial,
-    measure_fwhm,
-    measure_sharpness,
-    measure_centroid,
-    specFromSpectralType,
-    fwcentroid,
-)
-
-from .webbpsf_core import (  # noqa
-    instrument,
-    SpaceTelescopeInstrument,
-    JWInstrument,
-    NIRCam,
-    NIRISS,
-    NIRSpec,
-    MIRI,
-    FGS,
-)
-
-from .opds import enable_adjustable_ote  # noqa
-
-from .roman import WFI, RomanCoronagraph  # noqa
-
-from .match_data import setup_sim_to_match_file  # noqa
+os.environ["STPSF_PATH"] = os.environ.get("WEBBPSF_PATH", "")
```

Comment:
Oh this is very clever!  I had thought we would need code in `stpsf` to check WEBBPSF_PATH and if needed use that for STPSF_PATH. But doing it here as part of this import wrapper takes care of that so succinctly. I like this a lot.

Reply:
thanks!